1.1.c

```diff
@@ -0,0 +1,100 @@
+#include <string.h>
+#include <stdio.h>
+
+#define binarySize 256
+
+
+int transformation2ndWay(char* binary, char* octale)	//https://calcus.ru/perevod-sistem-schisleniya/iz-dvoichnoy-v-vosmerichnuyu here i took a way to transformation (2nd way)
+{
+int result = 0;
+int length = strlen(binary);
+if (length != 3) 
+{
+    return -1;
+}
+for (int i = 2; i >= 0; --i)
+{
+    if (binary[i] != '1' && binary[i] != '0')
+    {
+        return -1;
+    }
+    if (binary[2 - i] == '1')
+    {
+        int exponent = i;
+        int sum = 1;

```

Comment:
```suggestion
        int transformedBit = 1;
```
Это не сумма, это преобразованное значение бита двоичного числа
(-0.25)